fuse/eval/metrics/libs/classification.py

```diff
@@ -61,6 +61,44 @@ def auc_roc(
             y_score=y_score, y_true=np.asarray(target) == pos_class_index, sample_weight=sample_weight, max_fpr=max_fpr
         )
 
+    @staticmethod
+    def auc_roc_mult_label(
```

Comment:
multi_binary_label
Let's add the work binary next to multi_label to be more precise.